### PR TITLE
Move to golang ukifier

### PR DESCRIPTION
**Pull request by Itxaka**
Only has objcopy as dep

Signed-off-by: Itxaka <itxaka@kairos.io>

### CODE REVIEW

File: pkg/action/build-uki.go [hunk 1]
```diff
@@ -321,9 +360,11 @@ func (b *BuildUKIAction) createInitramfs(sourceDir, artifactsTempDir string) err
 		if err != nil {
 			return fmt.Errorf("getting record of %q failed: %w", filePath, err)
 		}
+		if rec.Name != strings.TrimPrefix(rec.Name, sourceDir) {
+			rec.Name = strings.TrimPrefix(rec.Name, sourceDir)
+		}
```
Comment (jimmykarily) [inline pkg/action/build-uki.go:365]:
```suggestion

		rec.Name = strings.TrimPrefix(rec.Name, sourceDir)

```

Comment (Itxaka) [reply pkg/action/build-uki.go:365]: let me try this with and without to see if it works and search on why I added it :D

Comment (Itxaka) [reply pkg/action/build-uki.go]: yep, works. Not sure why I though it would not lol

**Review by jimmykarily — APPROVED** [review]
Nice!

**Pull request merged**